benchmarks/bfv_zkp/src/bfv.rs

```diff
@@ -260,16 +260,9 @@ pub fn compile_proof() -> ZkpApplication {
 fn ark_bigint_to_native_field<B: ZkpBackend, F: MontConfig<N>, const N: usize>(
     x: Fp<MontBackend<F, N>, N>,
 ) -> NativeField<B::Field> {
-    assert!(N <= 8);
```

Comment:
@rickwebiii idk where your comment went, but I see it in my email notifications :shrug: will quote the relevant parts here:

> We should keep some assertion here, as converting a bigint is only correct if the Arkworks int <= 512 bit, regardless of the limb size.

This assert `N <= 8` still happens, it just happens in `ark_bigint_to_zkp_bigint`. (This code was doing the same thing so I just called the existing function.)

> Maybe:
> ```
> let max_limbs = if cfg!("32_bit") {
>   16
> } else if cfg!("64_bit) {
>   8
> } else {
>   unreachable!();
> }
> ```

We actually don't need to worry about that here, because these `N`s are used in `BigInt<N>`s which are always `[u64; N]` regardless of `target_pointer_width`. So, `N <= 8` iff `bitsize <= 512`